basilisk/src/ast/interpreter/test18.c

```
@@ -0,0 +1,37 @@
+/**
+Checks some math functions. **/
+
+int main()
+{
+  double c;
```

Clang (14.0.6)

[warning] variable 'c' is not initialized

(cppcoreguidelines-init-variables)